minard/templates/runselection.html

```diff
@@ -76,7 +76,7 @@ <h1 align="center">Run Selection</h1>
                 <th>
                     <select id="crit" onchange="get_limit(lim.value, {{offset}}, this.value, res.value, run.value, low.value, high.value, y_low.value, m_low.value, d_low.value, y_high.value, m_high.value, d_high.value);">
                     <option selected value="{{criteria}}">{{criteria}}</option>
-                    {% for n in ["scintillator", "partial_fill_antinu", "partial_fill", "water"] %}  <!-- FIXME: make this dynamic !-->
+                    {% for n in ["scintillator", "scintillator_silver", "scintillator_bronze", "directionality_analysis"] %}  <!-- FIXME: make this dynamic ?-->
```

Comment:
Does this need updating every time a new run criteria is created?

Reply:
Not quite, it only lists the most common criteria to make them easier to search by (others can be searched by typing in the URL). So it doesn't need to change any time we add any new criteria, but we might want to update it every now and again, like when we have loaded Te and create new criteria for that perhaps?

I could make this list something that can be changed in the settings file if you feel that would be better?

Reply:
I changed this to take the criteria for the drop-down menu from the settings file @tannerbk